src/styles/countdown.scss

```diff
@@ -0,0 +1,27 @@
+.countdown {
+  position: fixed;
+  width: 139px;
+  height: 174px;
+  right: 0px;
+  top: 154px;
+
+  background: $light-green;
+  border-radius: 15px 0 0 15px;
+  z-index: 1000;
+
+  &__text-wrapper {
```

Comment:
out of curiosity reasons for these specific margins?

Reply:
No reason, honestly, except that we had copied them straight from Figma. I just added another commit to tidy them up a bit and make them less arbitrarily specific.

Reply:
awesome I like that better :)